### ISSUE TITLE
[Build] Errors in fscomp.txt are not printed to the log

### ISSUE THREAD

**Issue by 0x53A** [dotnet/fsharp#3639]
Make FSComp.txt fail: just add something invalid, like ``xxxxxxxxxxxxx`` to the end

#### Actual:

```
1>------ Build started: Project: FSharp.Compiler.Private, Configuration: Debug Any CPU ------
1>Done building project "FSharp.Compiler.Private.fsproj" -- FAILED.
1>
1>Build FAILED.
========== Build: 0 succeeded, 1 failed, 1 up-to-date, 0 skipped ==========
```

#### Expected: any indication that there is an issue with FSComp. Even just printing a generic error "FScomp failed" would be better than nothing.

**Comment by smoothdeveloper** [comment]
* clone master
* duplicate a line in fscomp.txt
* call build.cmd

**Expected behavior**
> Build FAILED.
>    0 Warning(s)
>    1 Error(s).

**Actual behavior**

> snip (fscomp.txt is brought with mention of a duplicate)
> Build FAILED.
>    0 Warning(s)
>    0 Error(s)

**Known workarounds**

Comb the output and become more dubious about msbuild error reporting.

**Comment by cartermp** [comment]
Yeah this would be nice, https://github.com/dotnet/fsharp/pull/10035 is another example where having a space after a `,` just led to a nonsense build error.
